docs/source/api/v5/deliveryservice_requests_id_assign.rst

```diff
@@ -260,7 +260,7 @@ The response contains a full representation of the newly assigned :term:`Deliver
 			"innerHeaderRewrite": null,
 			"ipv6RoutingEnabled": null,
 			"lastHeaderRewrite": null,
-			"lastUpdated": "0001-01-01T00:00:00Z",
+			"lastUpdated": "2022-09-29T22:23:28-6:00",
```

Comment:
Do we not need milliseconds?

Reply:
This is the Http Post Body Data. We cannot add milliseconds in this case. This data is part of requested {} data that is returned to http operations.